aws/code/workflows/MLWorkflowMultiCPU.py

```diff
@@ -25,7 +25,7 @@
 
 from dask_ml.model_selection import train_test_split
 from sklearn.ensemble import RandomForestClassifier
-from sklearn.cluster import KMeans
+from dask_ml.cluster import KMeans
```

Comment:
Since we're planning to run these on single node there's an easier way to enable multi-core